src/store/conversationsStore.js

```diff
@@ -232,6 +264,14 @@ const mutations = {
 		if (unreadMentionDirect !== undefined) {
 			Vue.set(state.conversations[token], 'unreadMentionDirect', unreadMentionDirect)
 		}
+
+		// Update unread counters in the unreadCount store
+		this.dispatch('unreadCount/updateConversationCounts', {
+			token,
+			unreadMessages: state.conversations[token].unreadMessages || 0,
+			unreadMentions: state.conversations[token].unreadMention || 0,
+			unreadMentionDirect: state.conversations[token].unreadMentionDirect || 0,
```

Comment:
these two are either `true` or `0`. I'd treat them as boolean only. I wouldn't want to rely on `true - 0 === 1` and `0 - true === -1`